### DISCUSSION TITLE
Feature Request: Instructor mode (comments)

### DISCUSSION THREAD

**Discussion by slowizzm** [processing/p5.js-web-editor#2662] [General]
I would like to see a type of "instructor mode" implemented someday, a way to give instructors access to leave comments in the editor of a students sketch similar to the way Google Docs does commenting, only viewable to the student and instructor.

**Comment by catarak** [comment]
oooh, cool idea! i'm curious as to whether it makes sense to add a feature like this to the editor, or work on integration with existing tools. for example, working on #542 so that other services such as google classrooms can work well with the editor. still open and exploring this space!

**Comment by Laphatize** [comment]
I think this would be a really cool feature, I can definitely see schools using this!
